res/developer/builderConfig.json

```diff
@@ -0,0 +1,18 @@
+{
+  "appId": "com.electron.brave",
```

Comment:
I think this string (in each of the builderConfigs) should match what you have in `windowsInit.js`.

ex:
'com.squirrel.BraveNightly.BraveNightly' if nightly
'com.squirrel.BraveDeveloper.BraveDeveloper' if developer
'com.squirrel.BraveBeta.BraveBeta' if beta
'com.squirrel.brave.Brave' for all others

Reply:
might be good to search for the strings and make sure all occurrences look good 👍

Reply:
and my bad 😄  I totally missed these are not the same thing